app/src/main/java/uc/edu/klopfsea/groupscheduler/MainActivity.kt

```diff
@@ -21,7 +21,7 @@ class MainActivity : AppCompatActivity() {
 
     // creating empty ArrayList
     val eventList = arrayListOf<Event>()
-
+    @Override
```

Comment:
Ah, we use this in Java, but do we need it in Kotlin?  If so, please cite source.  We already have the override modifier in the method signature.